packages/FormElement/stories/examples/NativeInput.js

```diff
@@ -1,30 +0,0 @@
-import React from "react";
```

Comment:
Why remove a native element example? The way you apply certain attribute is different to using an `Input` component. Or are we now ok with them just spreading `a11yProps` on the `input` in general even though there are warnings

Reply:
Same warnings appear with `<Input>` since the `moreProps` go right onto the `<input>`.  I didn't see any real value in adding a native example since the usage is the same, and it's a very rare use case in our applications.  Certainly not one we would encourage as design system enforcers.